generator/templates/Authorisation/src/api/endpoints/authorisation/password.js

```diff
@@ -2,15 +2,15 @@ import { post } from '../../implementation/app';
 
 function passwordForgotten(email) {
```

Comment:
Dit was me al eerder opgevallen, maar in dit bestandje staan `passwordForgotten` en `passwordReset`, maar er bestaan ook twee bestanden genaamd `password/forgotten.js` en `password/reset.js`.

Ik zou de bestanden weghalen en dan het vervangen door deze methods. Anders wordt deze code niet eens gebruikt

Reply:
Dit bestand kon inderdaad weg